components/automate-ui/src/app/modules/infra-proxy/json-treeable/json-treeable.component.ts

```diff
@@ -0,0 +1,106 @@
+import { Component, Input, ElementRef, OnChanges } from '@angular/core';
+import * as jsonTree from '../../../../app/page-components/json-tree/vendor/json-tree';
```

Comment:
Convention is to avoid using `..` in import paths.
```suggestion
import * as jsonTree from 'app/page-components/json-tree/vendor/json-tree';
```